src/lando/main/models/profile.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import annotations
+
+from django.contrib.auth.models import User
+from django.db import models
+
+from lando.main.models.base import BaseModel
+
+
+class Profile(BaseModel):
+    user = models.OneToOneField(User, null=True, on_delete=models.SET_NULL)
+
+    # Phabricator API token.
+    phabricator_token = models.TextField(null=True, blank=True)
```

Comment:
Have we done an RRA for the auth changes? We probably should, especially since this change would store many Phab API tokens in plaintext in the Lando DB.

Reply:
No we have not, and this PR actually doesn't implement anything related to Phabricator API tokens, so I am going to remove this field from this PR. I'll schedule an RRA for this.

Also, in the long term, it would be great to automatically fetch (and possibly rotate) phabricator tokens. I'll file a bug for this as well.

Reply:
Removed the Phabricator API key and updated PR.